### PR TITLE
Fix #1661: In Register Activity. Validator code missing for Phone Number

### LINKED ISSUE #1661: In Register Activity. Validator code missing for Phone Number Field
**Summary:**

In Register Activity. Validator code missing for Phone Number Field

**Steps to reproduce:**

Go to Register Activity and try to register without entering phone number

**Expected behaviour:**

Warning should be displayed as shown for other fields

**Observed behaviour:**

No warning

**Device and Android version:**

Any

**Screenshots:**

### PR DESCRIPTION
**Pull request by ritish099**
Fixes #1661

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by ritish099** [comment]
@miPlodder Kindly check it out....

File: app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt [hunk 1]
```diff
@@ -179,6 +179,9 @@ class RegistrationFragment : BaseFragment(), RegistrationView {
         } else if (etLastName?.text.toString().trim { it <= ' ' }.isEmpty()) {
             Toaster.show(rootView, getString(R.string.error_validation_blank, getString(R.string.last_name)))
             return false
+        }else if (etPhoneNumber?.text.toString().trim { it <= ' ' }.isEmpty()) {
```
Comment (miPlodder) [inline app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt:181]: i see formatting is not perfect. `ctrl+alt+L` in your IDE before pushing

**Review by miPlodder — CHANGES_REQUESTED** [review]
Nice Job, format your code. rest looks fine

**Comment by ritish099** [comment]
@miPlodder reformatting does not change anything. I did some things manually kindly check it out.

**Comment by luckyman20** [comment]
The issue is already resolved.

**Pull request closed without merging**